installers/android-sdk/installer.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+YUM_CMD=$(which yum) # yum package manager for RHEL & CentOS
+DNF_CMD=$(which dnf) # dnf package manager for new RHEL & CentOS
+APT_GET_CMD=$(which apt-get) # apt package manager for Ubuntu & other Debian based distributions
+PACMAN_CMD=$(which pacman) # pacman package manager for ArchLinux
+APK_CMD=$(which apk) # apk package manager for Alpine
+
+ if [ ! -z $APT_GET_CMD ]; then
+    sudo apt-get update
+    sudo apt-get install android-sdk
+ elif [ ! -z $APK_CMD ]; then
+    sudo apk add --no-cache bash unzip libstdc++
+    mkdir -p /opt/android-sdk && cd /opt/android-sdk
+    && wget -q http://dl.google.com/android/repository/tools_r27.0.0-linux.zip -O android-sdk-tools.zip
+    && unzip -q android-sdk-tools.zip -d /opt/android-sdk
+    && rm -f android-sdk-tools.zip
+    && chmod 777 /opt/android-sdk/*
+    && echo y | android update sdk -a --no-ui --filter build-tools-25.0.2
+ elif [ ! -z $PACMAN_CMD ]; then
+    sudo pacman -S --needed base-devel git wget yajl
+    cd /tmp && git clone https://aur.archlinux.org/package-query.git
+    cd package-query/ && makepkg -si
+    cd /tmp/ && git clone https://aur.archlinux.org/yaourt.git
+    cd yaourt/ && makepkg si
+    yaourt -S android-sdk android-sdk-platform-tools android-sdk-build-tools
+    sudo touch /tmp/script.sh
+    cat <<EOF > /tmp/script.sh
+    export ANDROID_HOME=/opt/android-sdk
```

Comment:
We exporting these here is not permanent, right? I think we should be adding these to a permanent place, right?

Reply:
It will be permanent, I sourced the variable to the bash profile in the very next line

Reply:
This is how we set the environment variables for Java, maybe we can do the same.

```
cat >> /etc/environment <<EOL
JAVA_HOME=/usr/lib/jvm/java-8-oracle
JRE_HOME=/usr/lib/jvm/java-8-oracle/jre
EOL
```

Reply:
@agentmilindu yes, I read about this. Linux requires direct changes to the environment file. However, that would mean logging in as sudo. We should run the entire script as sudo then.

Reply:
No, We'll put the `sudo` command where necessary, the user will be prompted to enter the password then and there.